gnomad_qc/v5/data_ingestion/field_md_to_html.py

```diff
@@ -0,0 +1,115 @@
+import markdown
+from bs4 import BeautifulSoup
```

Comment:
I also just noticed this `bs4` import is causing pylint and docs errors -- I think we need to add this as a requirement for the repo